docs/Tutorials/BeginnersTutorial.md

```diff
@@ -0,0 +1,547 @@
+even venture into the code itself!
+
+## Prerequisites
+
+To start off, you'll need to obtain an environment to build and run SpECTRE in.
+You could try and install all the dependencies yourself, but that is very
+tedious and very error prone. Instead, we provide a
+[Docker](https://docs.docker.com/get-docker/) container with all the
+dependencies pre-installed for you to use. Probably the easiest way to get
+started with this is to follow our \ref dev_guide_quick_start_docker_vscode
+tutorial which will have you
+
+1. Clone SpECTRE
+2. Install Docker
+3. Install VSCode
+4. Open the repository in VSCode in the Development Container
+
+You can stop before the "Configure, compile and run SPECTRE" section as we will
+be going through that here as well, but with different and more in-depth
+examples.
+
+If you'd rather not use VSCode and instead just run the docker container in a
+terminal, you can follow the \ref docker_install tutorial in the \ref
+installation page.
+
+Another program you will need for this tutorial is [Paraview]
+(https://www.paraview.org/download/) for visualizing the output. If you are
+running any Debian-based OS (like Ubuntu), you can just install it via your
+package manager
+
+```
+sudo apt-get install paraview
+```
+
+Otherwise, you can just download the binary/executable. This is what you will
+need to do if you are running Windows or MacOS.
+
+
+## Building the ExportTimeDependentCoordinates3D Executable
+
+First, make sure you are inside the Docker container (whether this be in VSCode
+or just a terminal). First we will need to run `cmake` to set up everything
+because we use CMake as our build system.
+
+### With a Terminal {#with_terminal}
+
+Inside the SpECTRE repository, make a directory called `build`. This is where
+all the compiled objects will go. `cd` into the `build` directory. To configure
+SpECTRE in Release mode (which is recommended for running simulations), run the
+following command
+
+
+```
+cmake -D CMAKE_BUILD_TYPE=Release \
+      -D CHARM_ROOT=/work/charm_7_0_0/multicore-linux-x86_64-clang \
+      -D CMAKE_CXX_COMPILER=/usr/bin/clang++-10 \
+      -D CMAKE_C_COMPILER=/usr/bin/clang-10 ../
+```
+
+[Charm++](http://charmplusplus.org/) is the library we use for task-based
+parallelism. The `../` is the path to the root of the SpECTRE directory
+(relative or absolute, it doesn't matter). This should only take a few seconds
+to run.
+
+Now that everything is configured, you can build the
+`ExportTimeDependentCoordinates3D` executable like so
+
+```
+make ExportTimeDependentCoordinates3D -jN
+```
+
+Here, `N` is the number of cores you'd like to use to build. It's entirely up to
+you on how many you'd like to use. Since you are most likely running this on a
+laptop, `-j4` should be sufficient.
+
+### With VSCode
+
+If you are using VScode, open the
+[command palette]
+(https://code.visualstudio.com/docs/getstarted/userinterface#command-palette)
```

Comment:
Link problem again.

Fragment should be `_command-palette` (missing underscore).